lib/src/browser_client.dart

```diff
@@ -9,13 +9,14 @@ import 'dart:typed_data';
 import 'base_client.dart';
 import 'base_request.dart';
 import 'byte_stream.dart';
+import 'client.dart';
```

Comment:
[nit] May as well revert `lib/src/browser_client.dart`, `client_stub.dart`, and `io_client.dart`. The type changes aren't needed anymore with the `zoneClient` usage removed also.

Reply:
Done.